runtime/Go/antlr/utils.go

```diff
@@ -343,6 +341,33 @@ func EscapeWhitespace(s string, escapeSpaces bool) string {
 	return s
 }
 
+func TerminalNodeToStringArray(sa []TerminalNode) []string {
+	st := make([]string, len(sa))
+
+	for i, s := range sa {
+		st[i] = fmt.Sprintf("%v", s)
+	}
+
+	return st
+}
+
+func PrintArrayJavaStyle(sa []string) string {
+	var buffer bytes.Buffer
```

Comment:
Would it work to do something like `"[" + strings.Join(sa, ", ") + "]"`? It seems simpler to me.